packages/core/src/services/alert/service.ts

```diff
@@ -187,6 +187,7 @@ export class Service {
   ): { message: string; statusStr: string } {
     const badOracles = details?.badOracles?.join("\n");
     const noOracle = details?.noOracle;
+    const secondsToRecover = details?.secondsToRecover;
```

Comment:
It will just return an empty string.
``const secondsToRecoverStr =
      secondsToRecover && typeof secondsToRecover === "number"
        ? this.getSecondsToRecoverString(secondsToRecover)
        : "";```